src/app/components/TrainingsPage/TrainingsPage.tsx

```diff
@@ -2,115 +2,95 @@ import React from 'react';
+              {sectionTrainingsTitle as string}
+            </h2>
             <p className="trainings__text">
-              Our programs at AQE are designed to cater to the diverse needs of
-              educators at every stage of their career.
+              {sectionTrainingsDescription as string}
             </p>
           </div>
 
           <div className="cards">
-            <div className="cards__card cards__card--reverse">
-              <div className="cards__card__info" id="Trainings-malta">
-                <TrainingCard
-                  name="Malta"
-                  term1="28.07.2024 - 03.08.2024 "
-                  term2="04.08.2024 - 10.08.2024"
-                  duration="7 days"
-                  learningModule="yes"
-                  requiredLevel="Basic"
-                  link="#Home-form"
-                />
-              </div>
+            {trainingsCards.map((item, i) =>
+              i % 2 !== 0 ? (
+                <div key={`trainings-${i}`} className="cards__card test">
+                  <picture className="cards__card__imageContainer">
+                    <img
+                      className="cards__card__imageContainer-item"
+                      src="images/canary-desktop.jpg"
+                      alt="meeting-event"
+                    />
+                  </picture>
 
-              <picture className="cards__card__imageContainer">
-                <source
-                  media="(max-width:640px)"
-                  srcSet="images/malta-mobile.jpg"
-                />
-                <source
-                  media="(max-width:1200px)"
-                  srcSet="images/malta-tablet.jpg"
-                />
-                <img
-                  className="cards__card__imageContainer-item"
-                  src="images/malta-desktop.jpg"
-                  alt="meeting-event"
-                />
-              </picture>
-            </div>
-            <div className="cards__card">
-              <picture className="cards__card__imageContainer">
-                <source
-                  media="(max-width:640px)"
-                  srcSet="images/canary-mobile.jpg"
-                />
-                <source
-                  media="(max-width:1200px)"
-                  srcSet="images/canary-tablet.jpg"
-                />
-                <img
-                  className="cards__card__imageContainer-item"
-                  src="images/canary-desktop.jpg"
-                  alt="meeting-event"
-                />
-              </picture>
+                  <div className="cards__card__info" id="Trainings-canary">
+                    <TrainingCard
+                      value1={item.fields.field1Name as string}
+                      value2={item.fields.field2Name as string}
+                      value3={item.fields.field3Name as string}
+                      value4={item.fields.field4Name as string}
+                      value5={item.fields.field5Name as string}
```

Comment:
to wyglada strasznie wrzuc to w jakiegos arraya czy object i daj jako jeden props